src/careamics/lvae_training/eval_utils.py

```diff
@@ -596,51 +606,49 @@ def get_dset_predictions(
-                        nm_weight=model.loss_parameters.denoisplit_weight,
-                        gaussian_weight=model.loss_parameters.musplit_weight,
-                    )
-                    rec_loss = {"loss": rec_loss}  # hacky, but ok for now
-
-                # store rec loss values for first pred
-                if mmse_idx == 0:
-                    try:
-                        losses.append(rec_loss["loss"].cpu().numpy())
-                    except:
-                        losses.append(rec_loss["loss"])
+                # if loss_type == "musplit":
+                #     rec_loss = get_reconstruction_loss(
+                #         reconstruction=rec, target=tar, likelihood_obj=gauss_likelihood
+                #     )
+                # elif loss_type == "denoisplit":
+                #     rec_loss = get_reconstruction_loss(
+                #         reconstruction=rec, target=tar, likelihood_obj=nm_likelihood
+                #     )
+                # elif loss_type == "denoisplit_musplit":
+                #     rec_loss = reconstruction_loss_musplit_denoisplit(
+                #         predictions=rec,
+                #         targets=tar,
+                #         gaussian_likelihood=gauss_likelihood,
+                #         nm_likelihood=nm_likelihood,
+                #         nm_weight=model.loss_parameters.denoisplit_weight,
+                #         gaussian_weight=model.loss_parameters.musplit_weight,
+                #     )
+                #     rec_loss = {"loss": rec_loss}  # hacky, but ok for now
+
+                # # store rec loss values for first pred
+                # if mmse_idx == 0:
+                #     try:
+                #         losses.append(rec_loss["loss"].cpu().numpy())
+                #     except:
+                #         losses.append(rec_loss["loss"])
 
                 # update running PSNR
-                for i in range(num_channels):
-                    patch_psnr_channels[i].update(rec_img[:, i], tar[:, i])
+                # for i in range(num_channels):
+                #     patch_psnr_channels[i].update(rec_img[:, i], tar[:, i])
 
             # aggregate results
             samples = torch.cat(rec_img_list, dim=0)
             mmse_imgs = torch.mean(samples, dim=0)  # avg over MMSE dim
-            mmse_std = torch.std(samples, dim=0)
+            # mmse_std = torch.std(samples, dim=0)
             predictions.append(mmse_imgs.cpu().numpy())
-            predictions_std.append(mmse_std.cpu().numpy())
-
-    psnr = [x.get() for x in patch_psnr_channels]
-    return (
-        np.concatenate(predictions, axis=0),
-        np.concatenate(predictions_std, axis=0),
-        np.concatenate(logvar_arr),
-        np.array(losses),
-        psnr,
-    )
+            # predictions_std.append(mmse_std.cpu().numpy())
+
+    # psnr = [x.get() for x in patch_psnr_channels]
+    return np.concatenate(predictions, axis=0)
+    # np.concatenate(predictions_std, axis=0),
```

Comment:
If I remember correctly, this output is needed for the sake of calibration... So we might need to fix the code (as I guess it is commented out because there's some bug)

Reply:
But it is neither urgent nor necessary rn, so leaving a TODO is enough. And I also think all of this will be replaced by something @melisande-c has implemented.

Reply:
Yeah, commented because of some inconsistencies. We'd have to revisit